### ISSUE TITLE
Incorrect parsing of readme when checking for updates

### ISSUE THREAD

**Issue by MinionAttack** [BartoszCichecki/LenovoLegionToolkit#1411]
### Rules

- [X] I made myself familiar with the README.
- [X] I checked closed issues for similar ones and did not find a similar problem mentioned before.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

2.24.0

### OS

Windows 11 Pro 23H2 Build 22631.3958

### Device

Legion 7 16ARHA7

### BIOS version

K9CN46WW

### What's wrong?

Today a new update has appeared using the main repository (same as Lenovo Vantage) and when I click on readme, it doesn't parse properly and the HTML code appears.

![imagen](https://github.com/user-attachments/assets/96c09473-9d2e-41e4-ac17-88427c1d8406)

### How to reproduce the bug?

1. Go to Downloads.
2. Select Origin > Primary (Recomended)
3. Click on update.
4. Check for a Generic DisplayLink Driver for USB Docks and Adapters update.
![imagen](https://github.com/user-attachments/assets/4c36a36c-70d0-439b-9f03-a037e3e57161)
4.1 Link to the webiste: [https://support.lenovo.com/es/es/downloads/ds501908](https://support.lenovo.com/es/es/downloads/ds501908)
5. Click on open readme.

### What is the behavior that you expected?

The readme can be read in the window that appears or a browser is opened to view the HTML as it was done for [#1382](https://github.com/BartoszCichecki/LenovoLegionToolkit/issues/1382)

### Logs

N/A

### Additional information

Same problem in 2.23.0 (I just upgraded before opening this issue).

**Comment by BartoszCichecki** [comment]
Good catch. I modified the code to always open the README in a browser - this way I don't have to handle all the weird formats Lenovo is introducing.
